internal/mode/static/state/graph/certificate_bundle.go

```diff
@@ -0,0 +1,67 @@
+package graph
```

Comment:
Could you add a couple relevant unit tests for this file?

(TestValidateCA from configmaps_test.go could be moved here)

Reply:
will do